apps/client/src/components/VisualStyles.js

```diff
@@ -7,6 +8,26 @@ export const FigCaption = styled.figcaption`
 export const Figure = styled.figure`
   min-height: 1px; /* IE11 Bug */
   margin: 0 0 24px 0;
+  ${({ isLoading, theme, animateLoading, backgroundImage }) =>
+    isLoading
+      ? css`
+          ${perceivedLoading(animateLoading)({ theme })}
```

Comment:
Shameless copy/paste from `ImageCardStyle` from asc-ui :-P

Remove `animateLoading` and `theme`